src/argparse.cpp

```diff
@@ -54,7 +54,21 @@ bool ArgParse::parsingFails(Input& input, Output& output)
         string valueString;
         input >> valueString;
         // try to parse this token as a flag
-        while (optionalArgumentsRemaining && tryParseFlag(valueString)) {
+        while (optionalArgumentsRemaining) {
+            try {
+                if (!tryParseFlag(valueString)) {
+                    // if it is not a flag, then break, i.e
+                    // continue with positional args
+                    break;
+                }
+            }  catch (std::exception& e) {
```

Comment:
This branch is not covered according to the coverage report. Does it make sense to add a test for this?

Reply:
This code is not reachable at the moment because it is only executed if there is a command line flag with parameters in a command with position arguments. `list_clients` however is the onle command with parametrized flags, but it does not have positional args. Once this is merged, I plan to extend `foreach` by a `--name-match=…` flag, and then we can test this branch.

Reply:
Ok thanks! Just wanted to check :)